tests/tests-vanilla.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+current_directory=$(pwd)
+test_framework_directory="$current_directory/.jenkinsfile-runner-test-framework"
+working_directory="$test_framework_directory/work-vanilla"
+src_directory="$test_framework_directory/source"
+
+version="256.0-test"
+jenkinsfile_runner_tag="jenkins-experimental/vanilla-jenkinsfile-runner-test-image"
+
+. $test_framework_directory/init-jfr-test-framework.inc
+
+oneTimeSetUp() {
+  rm -rf "$working_directory"
+  mkdir -p "$working_directory"
+
+  cd "$src_directory"
+  result=$(docker build -t "$jenkinsfile_runner_tag" --no-cache . | grep 'Successfully tagged')
+  execution_should_success "$?" "$jenkinsfile_runner_tag" "$result"
+
+  cd "$current_directory"
+}
+
+test_scripted_pipeline() {
+  result=$(docker run --rm -v "$current_directory/test_resources/vanilla-images/test_scripted_pipeline":/workspace "$jenkinsfile_runner_tag" -a "param1=Hello" -a "param2=value2")
```

Comment:
Why not using the "run_jfr_docker_image" function? Are you missing any functionality?

Reply:
Because I want to add `-a "param1=Hello" -a "param2=value2"`. There is currently an opened PR in the test framework (https://github.com/jenkinsci/jenkinsfile-runner-test-framework/pull/15) to allow it.Doing it this way we can add checks to test https://github.com/jenkinsci/jenkinsfile-runner/pull/26, which is not tested

Reply:
Doesn't the changes introduced [here](https://github.com/jenkinsci/jenkinsfile-runner-test-framework/pull/13/files#diff-93253b70ce0abb644c69126e61809ff5R16) help? Or does the order affect?

Reply:
The order affect. "$3" is an option for the `docker run` command, not for the execution of the Jenkinsfile-runner